worker/master/master.go

```diff
@@ -149,17 +165,21 @@ func (m *Controller) IsLeader() bool {
 func (m *Controller) Start() error {
 	logrus.Debug("master controller starting")
 	start := func(ctx context.Context) {
+		pc := controller.NewProvisionController(m.kubeClient, &m.conf, map[string]controller.Provisioner{
```

Comment:
why create new?

Reply:
The informer in the ProvisionController uses stopCh, and it will be closed when exiting. If we don't create a new `pc`, it will panic 'close closed channel' when restarting